packaging/installer/methods/kickstart.md

```diff
@@ -94,22 +110,25 @@ should not need to use special values for any of these):
 
 ### Connect node to Netdata Cloud during installation
```

Comment:
This whole section is extremely bad. Connecting during installation just means copy/paste the kickstart command. Doing it after installation is exactly the same, unless for some reason you want to use the claiming script directly. It doesn't make sense. the way it is.

Reply:
This is the section I noted in the PR's comments that could be removed, if we give them a note about it earlier in the doc, do we need to repeat it again?

Reply:
Remove it then. Re-read both the quick start and this one from the beginning and see if they make sense/are compatible alone and together.